test/units/cli/test_galaxy.py

```diff
@@ -115,6 +117,32 @@ def test_display_galaxy_info(self):
         if display_result.find('\t\tgalaxy_tags:') > -1:
             self.fail('Expected galaxy_tags to be indented twice')
 
+    def test_execute_install(self):
+        # testing installing with insufficient information
+        gc = GalaxyCLI(args=["install"])
+        with patch('sys.argv', ["-c"]):
+            galaxy_parser = gc.parse()
+        self.assertRaises(AnsibleError, gc.run)
+
+        # installing role
```

Comment:
I would split these ('install' and 'install --offline -p -r etc') into two separate tests so any failures are more specific and granular.